### ISSUE TITLE
Accessibility issue: unable to get API and secret keys

### ISSUE THREAD

**Issue by shindere** [apache/cloudstack#3692]
I am a blind user of CloudStack and have to administer several domains.

Currently, I am unable to navigate the web interface to find out my
API and secret keys (e.g. to use CloudMonkey) because of accessibility
issues in the web interface. For instance at one stage I need to click on
an image but can't do so becuase it has no alternative text associated to
it.

Are you guys interested in immproving Cloudstack's accessibility?

Thanks,

**Comment by kioie** [comment]
Sounds like something I'd be willing to be a part of. Maybe you can share some of the things you'd like added to improve accessibility and overall experience

**Comment by shindere** [comment]
Thanks a lot for your interest!

Actually I think there's really a lot to do and I don't really know
where to start.

Just in case: will you come to FOSDEM? That may be a place for a real
meeting and then we can experiment together, that would be easier and
give you a good intuition of the problem, I think.

**Comment by kioie** [comment]
Not sure if I'll be at FOSDEM, but we can always setup a call and come up with a list of things we can do,  then we can share with the community as a starting point.

**Comment by shindere** [comment]
EK (2019/12/20 05:51 -0800):
> Not sure if I'll be at FOSDEM, but we can always setup a call and come
> up with a list of things we can do,  then we can share with the
> community as a starting point.

Sure, that's a good idea, thanks!

Perhaps let's see first whether you can make it to FOSDEM and fall back
to setting up a call if you can't?

**Comment by kioie** [comment]
Sure thing. Thanks.

**Comment by rhtyd** [comment]
@shindere this is addressed I think in both old UI and Primate. In Primate after you log in, you can go to your own user by clicking on top-right menu -> Profile there you can easily see your APIkey and secret key. If you've further idea please log UI related issues here: https://github.com/apache/cloudstack-primate/issues
